tests/e2e/test_running_cluster_monitoring_with_persistent_storage.py

```diff
@@ -57,13 +50,14 @@ class TestRunningClusterMonitoringWithPersistentStorage(E2ETest):
                  'alertmanager-main-2']
 
     @tier1
-    def test_running_cluster_mointoring_with_persistent_stoarge(self):
+    def test_running_cluster_mointoring_with_persistent_stoarge(
```

Comment:
'stoarge' - might as well fix this typo, no?

Reply:
done